autoload/peekaboo.vim

```diff
@@ -161,7 +163,7 @@ function! peekaboo#peek(count, mode, visualmode)
   return "\<Plug>(peekaboo)"
 endfunction
 
-function! peekaboo#aboo()
+function! peekaboo#aboo(insert_mode)
```

Comment:
This could be made `(...)` to use optional arg `a:0` instead of using `a:insert_mode`, if you think it's going to break some user function call.